src/views/CronTabList/CronTabRowActions.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { CronTabKind, cronTabModelRef } from '@crontab-model';
+import { useCronTabTranslation } from '@crontab-utils/hooks/useCronTabTranslation';
+import {
+  Dropdown,
+  DropdownItem,
+  DropdownPosition,
+  KebabToggle,
+} from '@patternfly/react-core';
+
+type CronTabRowActionsProps = {
+  obj?: CronTabKind;
+};
+
+const CronTabRowActions: React.FC<CronTabRowActionsProps> = ({ obj }) => {
+  const history = useHistory();
+  const { t } = useCronTabTranslation();
+  const [isDropdownOpen, setIsDropdownOpen] = React.useState(false);
+
+  const onEditCronTab = () => {
+    const cta = {
+      href: `/k8s/ns/${obj.metadata.namespace}/${cronTabModelRef}/${obj.metadata.name}/yaml`,
```

Comment:
I'm inclined to remove the actions from the initial PR until we can fix this properly. The problem is people will copy this sample code into their own plugins, and even if we update the sample, those real plugins won't get updated.

I'd suggest we get something working for the current released OpenShift versions and make sure they're in an appropriate release-* branch, then we can come back and add actions when the SDK functions we need are available.

Reply:
Makes sense. On the other side folks will check how other plugins are doing (constructing the URL manually) and use that.

But I like it more then having it here... will remove 👍